src/paperless_mail/mail.py

```diff
@@ -426,9 +428,21 @@ class MailAccountHandler(LoggingMixin):
 
     logging_name = "paperless_mail"
 
+    _message_preprocessor_types: list[type[MailMessagePreprocessor]] = [
+        MailMessageDecryptor,
+    ]
+
     def __init__(self) -> None:
         super().__init__()
         self.renew_logging_group()
+        self._init_preprocessors()
+
+    def _init_preprocessors(self):
+        self._message_preprocessors: list[MailMessagePreprocessor] = []
+        for preprocessor_type in self._message_preprocessor_types:
+            if preprocessor_type.able_to_run():
+                self._message_preprocessors.append(preprocessor_type())
+            self.log.debug(f"Skipping mail preprocessor {preprocessor_type.NAME}")
```

Comment:
```suggestion
            if preprocessor_type.able_to_run():
                self._message_preprocessors.append(preprocessor_type())
            else:
                self.log.debug(f"Skipping mail preprocessor {preprocessor_type.NAME}")
```

Assume this is what you meant